### PR TITLE
Update profile scores in line with the web

### PR DESCRIPTION
**Pull request by EVAST9919**
Screenshot_1

### CODE REVIEW

File: osu.Game/Overlays/Profile/Sections/Ranks/DrawableTotalScore.cs [hunk 1]
```diff
@@ -2,30 +2,260 @@
...
 namespace osu.Game.Overlays.Profile.Sections.Ranks
 {
-    public class DrawableTotalScore : DrawableProfileScore
+    public class DrawableTotalScore : OsuHoverContainer
     {
+        private const int height = 40;
+        private const int corner_radius = 6;
+
+        protected readonly Container InfoContainer;
+        protected readonly FillFlowContainer Accuracy;
```
Comment (peppy) [inline osu.Game/Overlays/Profile/Sections/Ranks/DrawableTotalScore.cs]: This should be called something else. You are using it to contain more than just accuracy.

**Review by peppy — CHANGES_REQUESTED** [review]

**Comment by peppy** [comment]
Why isn't this sharing implementation with `DrawableMostPlayedBeatmap`?

**Comment by EVAST9919** [comment]
> Why isn't this sharing implementation with `DrawableMostPlayedBeatmap`?

I thought it wasn't necessary because their mutual properties are only height and hover/idle colors

**Comment by peppy** [comment]
And corner radius, and box, no?

File: osu.Game/Overlays/Profile/Sections/DrawableUserInfoItem.cs [hunk 2]
```diff
@@ -0,0 +1,42 @@
...
+using osu.Framework.Allocation;
+using osu.Framework.Graphics;
+using osu.Framework.Graphics.Shapes;
+using osu.Game.Graphics;
+using osu.Game.Graphics.Containers;
+using System.Collections.Generic;
+
+namespace osu.Game.Overlays.Profile.Sections
+{
+    public class DrawableUserInfoItem : OsuHoverContainer
```
Comment (peppy) [inline osu.Game/Overlays/Profile/Sections/DrawableUserInfoItem.cs]: why isn't this called DrawableProfileRow or something? `UserInfoItem` tells me nothing about nothing name i could nothing.

File: osu.Game/Overlays/Profile/Sections/Ranks/DrawableWeightedScore.cs [hunk 3]
```diff
@@ -0,0 +1,39 @@
...
+            {
+                AutoSizeAxes = Axes.Y,
+                Width = 50,
+                Child = new OsuSpriteText
+                {
+                    Text = $"{Score.PP * weight:0}pp",
+                    Font = OsuFont.GetFont(weight: FontWeight.Bold, italics: true),
+                }
+            });
+            InfoContainer.Add(new OsuSpriteText
```
Comment (peppy) [inline osu.Game/Overlays/Profile/Sections/Ranks/DrawableWeightedScore.cs]: these names are still useless. `DetailInfo`? `InfoContainer`?









how about `LeftContent` / `RightContent` or something that makes sense to look at?

**Review by peppy — CHANGES_REQUESTED** [review]

**Comment by peppy** [comment]
How were you developing this? There's not even a test with `DrawableWeightedScore` as a compilable type.

**Comment by EVAST9919** [comment]
Superseded by https://github.com/ppy/osu/pull/7554

**Pull request closed without merging**